### ISSUE TITLE
doesn't run after updating to v0.10.1

### ISSUE THREAD

**Issue by kurokirasama** [mmabrouk/chatgpt-wrapper#301]
**Describe the bug**
I just updated to v0.10.1  via:
```
pip3 install git+https://github.com/mmabrouk/chatgpt-wrapper --upgrade
```
But running anything, even `chatgpt --version` gives this error:
```
Traceback (most recent call last):
  File "/home/kira/.local/bin/chatgpt", line 5, in <module>
    from chatgpt_wrapper.main import main
  File "/home/kira/.local/lib/python3.8/site-packages/chatgpt_wrapper/main.py", line 11, in <module>
    from chatgpt_wrapper.backends.api.repl import ApiRepl
  File "/home/kira/.local/lib/python3.8/site-packages/chatgpt_wrapper/backends/api/repl.py", line 8, in <module>
    from chatgpt_wrapper.backends.api.database import Database
  File "/home/kira/.local/lib/python3.8/site-packages/chatgpt_wrapper/backends/api/database.py", line 9, in <module>
    from chatgpt_wrapper.backends.api.schema.updater import SchemaUpdater
ModuleNotFoundError: No module named 'chatgpt_wrapper.backends.api.schema'
```

**To Reproduce**
Already describe

**Expected behavior**
open chatgpt

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04.6 LTS x86_64
 - Browser: chrome
 - Version v0.10.1

**Additional context**

**Comment by thehunmonkgroup** [comment]
Meh. Missed an `__init__.py` file for that dir.

Should be fixed in `0.10.2`

**Comment by kurokirasama** [comment]
> Meh. Missed an `__init__.py` file for that dir.

Thanks!

Now I'm getting this when running `chatgpt`:
```
WARNING: backend 'chatgpt-api' has been renamed to 'api', and support for the old name will be removed in a future release. Please update your config file to 
use 'api' for the 'backend:' setting.

SchemaUpdater - ERROR - Error during schema update process: Path doesn't exist: '/home/kira/.local/lib/python3.8/site-packages/chatgpt_wrapper/backends/api/schema/alembic'.  Please use the 'init' command to create a new scripts folder.
SchemaUpdater - ERROR - Stack trace: Traceback (most recent call last):
  File "/home/kira/.local/lib/python3.8/site-packages/chatgpt_wrapper/backends/api/schema/updater.py", line 84, in update_schema
    latest_version = self.get_latest_version()
  File "/home/kira/.local/lib/python3.8/site-packages/chatgpt_wrapper/backends/api/schema/updater.py", line 58, in get_latest_version
    script = ScriptDirectory.from_config(self.alembic_cfg)
  File "/home/kira/.local/lib/python3.8/site-packages/alembic/script/base.py", line 216, in from_config
    return ScriptDirectory(
  File "/home/kira/.local/lib/python3.8/site-packages/alembic/script/base.py", line 97, in __init__
    raise util.CommandError(
alembic.util.exc.CommandError: Path doesn't exist: '/home/kira/.local/lib/python3.8/site-packages/chatgpt_wrapper/backends/api/schema/alembic'.  Please use the 'init' command to create a new scripts folder.

An error occurred during the schema update process. Please check the logs
```
chatgpt config gives me the same
